client/src/main/java/com/github/gotify/client/ApiClient.java

```diff
@@ -297,13 +298,13 @@ public ApiClient setApiAuthorizations(Map<String, Interceptor> apiAuthorizations
     return this;
   }
 
-  public Retrofit.Builder getAdapterBuilder() {
-    return adapterBuilder;
+  public void setBaseUrl(String base) {
```

Comment:
This is generated code, I'd prefer not to change it, because when regenerating the client we would need to manually reapply these changes.

Reply:
Sorry but I can't find out how to update this generated client. I tried `./gradlew generateSwaggerCode` but Android Studio kept complaining that `client` is not a gradle project and refuse to build until I reset the whole repo.

Reply:
Maybe we can update the definition with the gotify server release

Reply:
Jup, but we don't have to because we don't need any of the new functionality in gotify/android (yet).